sdk/storage/azfile/internal/testcommon/clients_auth.go

```diff
@@ -106,6 +108,10 @@ func GetGenericAccountInfo(accountType TestAccountType) (string, string) {
 	}
 	accountNameEnvVar := string(accountType) + AccountNameEnvVar
 	accountKeyEnvVar := string(accountType) + AccountKeyEnvVar
+	if accountType == TestAccountPremium {
```

Comment:
AccountNameEnvVar & this new one, why do we have two env variables ?

Reply:
This is for the premium file shares account, where the name of the environment variables in the test resources are `PREMIUM_FILE_STORAGE_ACCOUNT_NAME` and `PREMIUM_FILE_STORAGE_ACCOUNT_KEY`.